G2P.pm

```diff
@@ -1070,7 +1070,7 @@ sub _exceeds_frequency_threshold {
 sub _vcf_frequency_filtering {
   my $self = shift;
   my $tva = shift;
-  my $allele = $tva->variation_feature_seq;
+  my $allele = $tva->base_variation_feature->alt_alleles;
```

Comment:
The `$allele` is an array ref, line 1079 compares this `$allele` with the string `$_->allele`:
`my @alleles = grep {$_->allele eq $allele} @{$vcf_collection->get_all_Alleles_by_VariationFeature($vf)};`
this comparison never returns results.
You have to change the way the alleles are being compared.

Reply:
This issue is still ongoing.

Reply:
When running with the option `af_from_vcf=1` the undef allele causes missing frequencies in the report.

Current code:
P1	ENSG00000174842(GLMN,VMGLOM HGNC:14373;  G2P confidence category:definitive)	ENST00000495106	not_canonical	OBS=monoallelic_autosomal	REQ=monoallelic_autosomal	1:92266456:92266456:T:A:HOM:missense_variant,NMD_transcript_variant:SIFT=deleterious(0.0):PolyPhen=benign(0.361):gnomADe=4.012e-06,gnomADe:ALL=4.012e-06,gnomADe:nfe=8.86e-06,gnomADe_NFE=8.86e-06

New code:
P1	ENSG00000174842(GLMN,VMGLOM HGNC:14373;  G2P confidence category:definitive)	ENST00000495106	not_canonical	OBS=monoallelic_autosomal	REQ=monoallelic_autosomal	1:92266456:92266456:T:A:HOM:missense_variant,NMD_transcript_variant:SIFT=deleterious(0.0):PolyPhen=benign(0.361):gnomADe=4.012e-06,gnomADe_NFE=8.86e-06